cirq/circuits/circuit.py

```diff
@@ -763,16 +763,16 @@ def _can_commute_past(self,
     def insert(
             self,
             index: int,
-            moment_or_operation_tree: Union[Moment, ops.OP_TREE],
+            operations: ops.OP_TREE,
```

Comment:
I think the moment or operation tree name is better, because it indicates to the user that moments are handled in a special way.